sr_error_reporter/README.md

```diff
@@ -0,0 +1,23 @@
+# sr_error_reporter
+
+A package for calculating and publishing errors.
+
+This node can be started by:
+```
+rosrun sr_error_reporter error_reporter
+```
+
+## Underactuation error
+
+Underactuation error is calculated by listening to topic `/joint_states` for
+actual values from sensors and by listening to topics `/lh_trajectory_controller/command`
+and `/lh_trajectory_controller/command` for desired values.
```

Comment:
/rh_trajectory_controller/command

Reply:
Well spotted! Fixed in 80091116827c6404158c8d71d1bc4903c4906bf5.